Cargo.toml

```diff
@@ -3,14 +3,14 @@ name = "zram-generator"
 version = "0.1.2"
 authors = ["Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>"]
 license = "MIT"
-description = "Systemd unit generator for zram devices"
+description = "Systemd unit generator for zram swap devices"
 homepage = "https://github.com/systemd/zram-generator"
 edition = "2018"
 
 [dependencies]
-anyhow = "1"
+anyhow = "1.0"
```

Comment:
hm, what is the reason for this change?

Reply:
No idea. But it shouldn't be part of this patch even if it is otherwise justified, so I dropped it.

Reply:
Right, I had this kicking about in my Cargo.toml and missed it; soz.